### PR TITLE
Cypress/testing install main page

### PR DESCRIPTION
**Pull request by bhohnco**
## Type of Change
- [ ] JSX
- [ ] HTML
- [ ] JavaScript
- [ ] CSS
- [ X ] React
- [ ] React Router
- [ X ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [ ] Refactor/Fix Bugs

## Please describe the change made
Cypress has been installed

## Please give a short description of the changes that were made below

- Fix:

- Feature:
Cypress testing has been added. 
10 cypress tests total
The main-view page and form tests are added and are passing. 

## How has this been tested?
- [ ] In Browser
- [ ] In Console
- [ X ] Cypress
- [ ] Postman

### CODE REVIEW

File: walk-safe/cypress/integration/mainpage.spec.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+describe('Show main view of walk-safe App', () => {
+
```
Comment (carolineshowscode) [inline walk-safe/cypress/integration/mainpage.spec.js:2]: Once we get our BE connected here is some documentation for intercepting GraphQL calls: https://docs.cypress.io/guides/testing-strategies/working-with-graphql#Alias-multiple-queries-or-mutations

**Review by carolineshowscode — APPROVED** [review]
Nice job testing main page view and inputs! I added a note on GraphQL Cypress testing for our queries and mutations

File: walk-safe/cypress/integration/mainpage.spec.js (hunk 1 above)
Comment (pcmueller) [reply walk-safe/cypress/integration/mainpage.spec.js:2]: Thx Caroline - will dig into the docs then

**Review by pcmueller — APPROVED** [review]
Thanks, solid basis for testing!

**Review by foymikek — APPROVED** [review]
Nice additions, code looks super clean!

**Pull request merged**